.gitignore

```diff
@@ -140,3 +140,4 @@ ehthumbs.db
 Thumbs.db
 
 *.user.2.7pre1
+editDoc.py
```

Comment:
please remove this. instead of adding to the gitignore, just don't commit this change